models/spring-ai-ollama/src/main/java/org/springframework/ai/ollama/OllamaChatModel.java

```diff
@@ -460,7 +460,8 @@ else if (message instanceof ToolResponseMessage toolMessage) {
 		OllamaApi.ChatRequest.Builder requestBuilder = OllamaApi.ChatRequest.builder(requestOptions.getModel())
 			.stream(stream)
 			.messages(ollamaMessages)
-			.options(requestOptions);
+			.options(requestOptions)
+			.think(requestOptions.getThink());
```

Comment:
This seems awkward, should be part of the `.options` call.  Similar to my other comment in the Builder class.